core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -2344,16 +2249,18 @@ class LogTest {
   @Test
   def shouldNotDeleteSegmentsWhenPolicyDoesNotIncludeDelete() {
     def createRecords = TestUtils.singletonRecords("test".getBytes, key = "test".getBytes(), timestamp = 10L)
-    val log = createLog(createRecords.sizeInBytes,
-      retentionMs = 10000,
-      cleanupPolicy = "compact")
+    val logProps = new Properties()
+    logProps.put(LogConfig.SegmentBytesProp, createRecords.sizeInBytes * 5: Integer)
+    logProps.put(LogConfig.RetentionMsProp, 10000: Integer)
+    logProps.put(LogConfig.CleanupPolicyProp, "compact")
```

Comment:
It seems like we could introduce a `logProps` method to keep things concise. Something along the lines of:

```scala
def logProps(messageSizeInBytes: Int, retentionMs: Int = -1, retentionBytes: Int = -1, ...)
    val logProps = new Properties()
    logProps.put(LogConfig.SegmentBytesProp, messageSizeInBytes * messagesPerSegment: Integer)
    logProps.put(LogConfig.RetentionMsProp, retentionMs: Integer)
    logProps.put(LogConfig.RetentionBytesProp, retentionBytes: Integer)
    logProps.put(LogConfig.CleanupPolicyProp, cleanupPolicy)
    logProps.put(LogConfig.MessageTimestampDifferenceMaxMsProp, Long.MaxValue.toString)
    logProps
}
```

What do you think?